tests/test_runs/test_run_functions.py

```diff
@@ -361,17 +412,75 @@ def determine_grid_size(param_grid):
             self.assertTrue(check_res)
 
         # todo: check if runtime is present
-        self._check_fold_evaluations(run.fold_evaluations, 1, num_folds)
+        self._check_fold_evaluations(run.fold_evaluations, 1, num_folds,
+                                     task_type="Supervised Classification")
+        pass
+
+    def _run_and_upload_regression(self, clf, rsv):
```

Comment:
I think it would be great if you could rename that other function to `_run_and_upload_classification`.